Competition2017/src/competition/subsystems/feeder/commands/SetFeederSpeedCommand.java

```diff
@@ -0,0 +1,32 @@
+package competition.subsystems.feeder.commands;
+
+import competition.subsystems.feeder.SideFeederSubsystem;
+import xbot.common.command.BaseCommand;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class SetFeederSpeedCommand extends BaseCommand {
+
+    final SideFeederSubsystem sideFeeder;
+    final DoubleProperty commandedFeederSpeed;
+
+    public SetFeederSpeedCommand(SideFeederSubsystem sideFeeder, XPropertyManager propertyManager) {
+        
+        this.sideFeeder = sideFeeder;
+        this.requires(this.sideFeeder);
+
+        commandedFeederSpeed = propertyManager.createEphemeralProperty("Commanded feeder speed", 3100.0);
```

Comment:
This feels wrong. Why is this property being used for the speed? Shouldn't it be a parameter?